### PR TITLE
add older meetups

### LINKED ISSUE #3: Need to add (a) page(s) about meetups

### PR DESCRIPTION
**Pull request by MBrouns**
fixes #3

### CODE REVIEW

**Comment by JCZuurmond** [comment]
Yes, looks good to me!

File: _posts/meetups/2019-02-26-nlp_in_practice.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 ---
```
Comment (clamm) [inline _posts/meetups/2019-02-26-nlp_in_practice.md]: The date in the file name should be 2019-12-11 (if the date from the file is picked up for the menu item)

Comment (MBrouns) [reply _posts/meetups/2019-02-26-nlp_in_practice.md]: Yep, also fixed the meetups that said they were in 2018

**Comment by Nelson-AM** [comment]
The sublist of old meetups still has horizontal dividers between each entry. I'd like to see that only for the top list, but I haven't figured out precisely how to fix this yet.

This will also be useful for the older conferences sublist.

**Comment by MBrouns** [comment]
Would indeed look better. I think we can get there by adding a direct child selector somewhere as the meetup sublist has its own `<ul>` group. That said though, the current 'historical meetup' makes no sense so I do think this PR is an improvement even with the dividers. Should we pick up the dividers in a separate PR?

**Comment by MBrouns** [comment]
NVM, got rid of it!

image

**Review by Nelson-AM — APPROVED** [review]

**Pull request merged**